src/SmartComponents/DriftPage/DriftPage.js

```diff
@@ -15,7 +15,7 @@ class DriftPage extends Component {
     render() {
         return (
             <React.Fragment>
-                <PageHeader>
+                *<PageHeader>
```

Comment:
Is the `*` intentional?

Reply:
nope, was a mistake! I was wondering where that came from...